opsimulator/opsimulator.go

```diff
@@ -168,6 +174,9 @@ func (opSim *OpSimulator) startBackgroundTasks() {
 				}
 
 				opSim.log.Info("OptimismPortal#depositTransaction", "l2TxHash", depTx.Hash().String())
+				if err := opSim.l1DepositStoreManager.Set(l1Log.TxHash, dep); err != nil {
```

Comment:
a single l1 transaction can have multiple deposit tx inside it so it cannot be used as the index. instead the deposit tx's own transaction hash might be a better one

Reply:
got it makes sense updating this